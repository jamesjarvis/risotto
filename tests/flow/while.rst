for i := 2; i < 10; i = i+1  {
    println(i)
}

for i := 0; i < 3; i = i+1 {
    for j := 18; j < 20; j = j+1 {
        println(i + " " + j)
    }
}

i := 0
while i < 10 {
    println(i)
    i=i+1
}

println("done")
