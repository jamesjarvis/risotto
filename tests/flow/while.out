2
3
4
5
6
7
8
9
0 18
0 19
1 18
1 19
2 18
2 19
0
1
2
3
4
5
6
7
8
9
done
